ludwig/api.py

```diff
@@ -456,6 +458,9 @@ def train(
         :param random_seed: (int, default: `42`) a random seed that will be
             used anywhere there is a call to a random number generator: data
             splitting, parameter initialization and training set shuffling
+        :param save_ludwig_config_with_model_weights: (bool, default: False) indicates
+            whether the user-provided ludwig-config should be saved along with
+            the model weights (to be eventually uploaded to HF)
```

Comment:
@sanjaydasgupta Here is another thought.  Could you please generate the documentation -- it is in the `ludwig-docs` repository and see if your new flag shows up properly?  The `README` in that repository provides instructions for how to run `mkdocs serve` and see the actual rendering.  Thank you very much.